auth/src/config/index.ts

```diff
@@ -1,13 +1,23 @@
-import fs from 'fs';
-
 export namespace Config {
     export const DATABASE_PATH = 'database/';
     export const DATASTORE_READER = getReaderUrl();
+    export const DATASTORE_WRITER = getWriterUrl();
 
     function getReaderUrl(): string {
-        if (!process.env.DATASTORE_READER_HOST || !process.env.DATASTORE_READER_PORT) {
+        const readerHost = process.env.DATASTORE_READER_HOST;
+        const readerPort = process.env.DATASTORE_READER_PORT;
+        if (!readerHost || !readerPort) {
             throw new Error('No datastore reader is defined.');
         }
-        return `http://${process.env.DATASTORE_READER_HOST}:${parseInt(process.env.DATASTORE_READER_PORT, 10)}`;
+        return `http://${readerHost}:${parseInt(readerPort, 10)}`;
+    }
+
+    function getWriterUrl(): string {
```

Comment:
If you do not need the writer in non-test modes, do not include writercode in the normal code and move it to the tests.